core/statistics/tpsBenchmark_test.go

```diff
@@ -636,3 +636,47 @@ func TestTpsBenchmark_ZeroTxMetaBlockAndEmptyShardHeader(t *testing.T) {
 	bigTxCount := big.NewInt(0)
 	assert.Equal(t, bigTxCount, tpsBenchmark.TotalProcessedTxCount())
 }
+
+func TestTpsBenchmark_ShardStatisticConcurrentAccess(t *testing.T) {
+	t.Parallel()
+
+	tpsBenchmark, _ := statistics.NewTPSBenchmark(12, 4)
```

Comment:
use recover() as you should check that no panic has encountered during the test

Reply:
done.